src/support_sphere_db/src/ssdb/models/user_profile.py

```diff
@@ -0,0 +1,13 @@
+import uuid
+from sqlmodel import Field, SQLModel
+
+
+class UserProfile(SQLModel, table=True):
+    __tablename__ = "user_profiles"
+
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
```

Comment:
Are you able to find a way to connect this with the `auth.user.id`
field that's managed by supabase? This should be a foreign key as well as primary key.